roles/paperless_ng/defaults/main.yml

```diff
@@ -0,0 +1,28 @@
+---
+paperless_ng_enabled: false
+
+# directories
+paperless_data_directory: "{{ docker_home }}/ng_paperless"
```

Comment:
Var prefixes should be consistent - typical standard is that they match the role name as you have done with `paperless_ng_enabled`.

Directory names should match the role name too, `ng_paperless` feels backwards.